### PR TITLE
Implement option to search movie by year as well

### PR DESCRIPTION
**Pull request by ndedic**
Modifies entry point to cli by passing complete argv to it so it's more flexible to adding different query params (e.g. year)

### CODE REVIEW

File: src/imdbtr.js [hunk 1]
```diff
@@ -4,15 +4,21 @@ const chalk = require('chalk');
...
+
+  const query = helpers.buildQuery(params);
+  const movie = api(query);
 
   if (!movie) {
     return false;
   }
 
-  const spinner = ora(`Searching for ${chalk.yellow(name)}`).start();
+  const spinner = ora(`Searching for ${chalk.yellow(query)}`).start();
```
Comment (fernahh) [inline src/imdbtr.js]: can you improve this information? I've think that "Searching for ${title} from ${year}" is better...

File: bin/imdbtr [hunk 2]
```diff
@@ -11,10 +11,12 @@ const usage = `
 	${chalk.black.bgYellow.bold(`imdbtr`)} - IMDb on terminal
 
 	Usage
-	  $ imdbtr <movie>
+		$ imdbtr <movie>
```
Comment (fernahh) [inline bin/imdbtr]: Replace with options:

```
Usage
  $ imdbtr [OPTIONS]

Example:
  $ imdbtr --year 1972

Options:
  -y --year            Filter movie by year
  -h --help            Display usage details
  
```

**Review by fernahh — CHANGES_REQUESTED** [review]
Hey @ndedic! I just made some comments. 

Thanks so much! 🎉

File: src/imdbtr.js (hunk 1 above)
Comment (ndedic) [reply src/imdbtr.js]: Sure. the only problem is if a different set of arguments is used (other than `title` and `year`) so we can look to implement a generic message e.g. "Searching for: key1 = value1 and key2 = value2 and ..." or something similar

Comment (fernahh) [reply src/imdbtr.js]: maybe "Searching for value1, value2", just separated with commas.

**Comment by fernahh** [comment]
#10

File: src/imdbtr.js (hunk 1 above)
Comment (ndedic) [reply src/imdbtr.js]: ok

Comment (ndedic) [reply src/imdbtr.js]: I'll fix later today

**Comment by fernahh** [comment]
@ndedic 

tip: [add you e-mail on `.gitconfig` file](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup). this will avoid the anonymous commits.

**Review by fernahh — APPROVED** [review]

**Comment by fernahh** [comment]
Thanks @ndedic! :tada:

**Comment by ndedic** [comment]
np

**Pull request merged**